tests/workflows/test_cluster_plots.py

```diff
@@ -0,0 +1,49 @@
+import json
+import os
+
+import luigi
+import pytest
+from google.cloud import storage
+
+from workflows.cluster_plots.__main__ import ClusterPlotTaskWrapper
+
+default_output_path = "/home/nzhon/data/processed/birdclef-2022/birdnet-embeddings-with-neighbors-static/v1"
+
+
+def test_cluster_plots_task_luigi_build(output_path=default_output_path):
+    f = open(
+        "/home/nzhon/data/processed/birdclef-2022/birdnet-embeddings-with-neighbors-agreement-static/v1/agreement.json"
```

Comment:
Take a few rows of this dataset and stick it into the test directory as a fixture. Ideally we want to be able to run this test on every PR.